cosmo_tester/framework/test_hosts.py

```diff
@@ -814,9 +794,19 @@ def __init__(self,
 
         if bootstrappable:
             for instance in self.instances:
-                instance._image_name = ATTRIBUTES['default_linux_image_name']
+                instance._image_name = self.bootstrappable_image_name
                 instance.should_finalize = False
 
+    @property
+    def bootstrappable_image_name(self):
+        image_name = ATTRIBUTES['cloudify_manager_installer_image_name']
```

Comment:
Shouldn't this auto-generate the image name the way the manager image does (when the attribute isn't populated)?

Then you could just do a get on the attribute and not worry about exceptions on Jenkins.

Reply:
Well, you are right for this and should be update once we sync the community version + premium version and thats the reason why I added the exception there and added a note about that